src/sst/core/serialization/impl/serialize_insertable.h

```diff
@@ -189,19 +193,20 @@ class serialize_impl<
                         typename OBJ::key_type key {};
                         SST_SER(key);
                         auto& value = obj[std::move(key)];
-                        SST_SER(value);
+                        SST_SER(value, opts);
                     }
                     else if constexpr (
                         is_same_template_v<T, std::multimap> || is_same_template_v<T, std::unordered_multimap> ) {
                         typename OBJ::key_type key {};
                         SST_SER(key);
                         auto& value = obj.emplace(std::move(key), typename OBJ::mapped_type {})->second;
-                        SST_SER(value);
+                        SST_SER(value, opts);
                     }
                     else if constexpr (
                         is_same_template_v<T, std::set> || is_same_template_v<T, std::unordered_set> ||
                         is_same_template_v<T, std::multiset> || is_same_template_v<T, std::unordered_multiset> ) {
                         typename OBJ::key_type key {};
+                        // TODO: Figure out how to make as_ptr_elem work with sets
```

Comment:
Checking: this is a future todo?

Reply:
Yes.  It will require a change to the serializer API.  For all other container types, you can default create the object in the container and then deserialize it in-place.  You can't do that with sets since you can't insert until it has been deserialized since it contains keys only.  We'll need to add a way to rewrite the pointer map table before we can make sets work.